apache/submarine
# SUBMARINE-1010. Use kubectl wait for the minio pod in mlflow

### What is this PR for?
Use init container to wait for the minio pod start. Add this container to mlflow and server pod. Add the rbac credential to mlflow pod.

### What type of PR is it?
Improvement

### Todos


### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-1010

### How should this be tested?
See the following video. It shows the logs of server and mlflow pod and the two pods run successfully. 
https://youtu.be/T1eSeyeyXvk

### Screenshots (if appropriate)


### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No
